website_sale_product_item_cart_custom_qty/tests/test_website_sale_product_item_cart_custom_qty.py

```diff
@@ -15,6 +15,4 @@ def setUp(self):
 
     def test_ui_website(self):
         """Test frontend tour."""
-        self.start_tour(
-            "/shop", "website_sale_product_item_cart_custom_qty", login="admin"
-        )
+        self.start_tour("/", "website_sale_product_item_cart_custom_qty", login="admin")
```

Comment:
Why are you changing the start url?

Reply:
Is it necessary if it is already being told in the tour.register where the tour will run? because it was giving a tour start error when I did the test locally

Reply:
Can you try running it here without that change?

Reply:
Done, the problem is solved here, thanks!